### ISSUE TITLE
Linter complain on syntax to call `create2`

### ISSUE THREAD

**Issue by mfornet** [duaraghav8/Ethlint#294]
**Description**
It should pass but complains about syntax error during linting:

```
contracts/test/StateTest.sol
  22:48    error    Syntax error: unexpected token {
```

For the code:

```solidity
contract SelfDestructFactory {
    constructor() { }

    function deploy() public returns(address) {
        address addr = address(new SelfDestruct{salt: bytes32(uint256(0x1234))}());
        return addr;
    }
}
```

**Steps to reproduce**
This section may contain all or a combination of the following:

- The command you used to lint
- The solidity code over which you ran the linter
- Contents of your `.soliumrc.json` file
- Contents of your `.soliumignore`

In addition, you may add anything you feel is relevant.

**Expected behavior**

**Operating System**
OS you ran the linter on (Windows / MacOS / Linux / Any other)

**Linter version**
The output of `solium --version`

**Comments**
If you'd like to share any additional information that doesn't fit in above sections or suggest a solution to the maintainers.

**Priority**
If you'd like to, specify the priority of this bug for your organisation. Priority could be `High` (meaning that the bug is seriously hindering your workflow), `Medium`, or `Low`.
